packages/mco/utils/disaster-recovery.tsx

```diff
@@ -301,20 +310,36 @@ export const matchClusters = (
     decisionClusters?.includes(drClusterName)
   );
 
+const getScheduledSynInterval = (scheduledSyncInterval: string) => {
+  const regex = new RegExp(
+    `([0-9]+)|([${TIME_UNITS.Days}|${TIME_UNITS.Hours}|${TIME_UNITS.Minutes}]+)`,
+    'g'
+  );
+  const splittedArray = scheduledSyncInterval?.match(regex);
+  const interval = Number(splittedArray?.[0] || 0);
+
+  const unit = splittedArray?.[1] || 's';
```

Comment:
nit: we are supporting m/d/h... so default should be one of those...
```suggestion
  const unit = splittedArray?.[1] || 'm';
```